webapp/components/CommentList/CommentList.vue

```diff
@@ -1,12 +1,12 @@
 <template>
   <div id="comments" class="comment-list">
     <h3 class="title">
-      <counter-icon icon="comments" :count="postComments.length" />
+      <counter-icon icon="comments" :count="commentsCount" />
       {{ $t('common.comment', null, 0) }}
     </h3>
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-08-04T13:23:43Z" title="Tuesday, August 4th 2020, 3:23:43 pm +02:00">Aug 4, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,7 +1,10 @@
 <template>
   <div id="comments" class="comment-list">
     <h3 class="title">
-      <counter-icon icon="comments" :count="postComments.length" />
+      <counter-icon
+        icon="comments"
+        :count="postComments.filter((comment) => !comment.deleted).length"
```


I looked at the other place in `webapp/components/PostTeaser/PostTeaser.vue` where the counter is set on a post teaser. This is the part which differs from that here.
Because I know this value of `post.commentsCount` comes direct from the backend and because I have found this doesn’t count `comment.deleted` or `comment.disabled` I will suggest this solution to be taken over here.

```suggestion
        :count="post.commentsCount“
```

A test in `backend/src/schema/resolvers/posts.spec.js` is missing which checks that only comments not deleted and not disabled are counted by the Cypher queries.
If you are motivated would be nice to implement this one. 💫🍀

Reply:
> [<img alt="Elweyn" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Elweyn) **Authored by [Elweyn](https://github.com/Elweyn)**
_<time datetime="2020-08-05T07:58:44Z" title="Wednesday, August 5th 2020, 9:58:44 am +02:00">Aug 5, 2020</time>_

---


Thanks a lot, this one seems a lot better than a filter on the deleted comments!
I will implement this so the frontend code stays clean of logic.